pkg/review/commit_review_status.go

```diff
@@ -152,9 +153,9 @@ type BigQueryBreakGlassIssueFetcher struct {
 	config *CommitApprovalPipelineConfig
 }
 
-func (bqif *BigQueryBreakGlassIssueFetcher) getBreakGlassIssues(ctx context.Context, author, timestamp string) ([]breakGlassIssue, error) {
-	issueQuery := GetBreakGlassIssueQuery(bqif.config.IssuesTable, author, timestamp)
-	items, err := bq.Query[breakGlassIssue](ctx, bqif.client, issueQuery)
+func (bqif *BigQueryBreakGlassIssueFetcher) getBreakGlassIssues(ctx context.Context, author, timestamp string) ([]*breakGlassIssue, error) {
```

Comment:
Does pointerifying this make much of a difference? I would assume the only real difference is allocating a lot of smaller memory regions rather than one contiguous one. Depending on existing fragmentation of memory, my mental model would make it seem like []breakGlassIssue would be faster, as it should be fewer allocs (as long as array is pre-allocated rather than appended)

Reply:
In the grand scheme of "things that are so big we need to run in Beam", ¯\_(ツ)_/¯. But local testing showed a pretty consistent 2s improvement.